i18n/others/en-US.json

```diff
@@ -41,6 +41,39 @@
   "Torpedo": "Torpedo",
   "AntiAir": "AntiAir",
   "Armor": "Armor",
-  "Luck": "Luck"
+  "Luck": "Luck",
+
+  "doyouknow-prefix": "#Tip# ",
+  "doyouknow-contents": [
+    "We frequently introduces new features in \"Tips\". You can disable it in settings though",
+    "We look forward to your suggestions! Please post it on our GitHub repo: poooi/poi",
+    "Hang round the settings and plugin list regularly. You may find new features!",
+    "Anti-catbomb (network retry) ALWAYS comes at risk. USE WITH CAUTIOUS!",
+    "poi never modifies or actively sends network packets, provided you use trusted poi and plugins",
+    "Switch between \"Overview\" and \"Fleet\" using Ctrl/Cmd+1/2",
+    "Switch between plugins using Ctrl/Cmd+3/4/5/6/7/8",
+    "Switch between plugins sequentially using Tab",
+    "\"Overview\" and \"Fleet\" can also be switched with double click!",
+    "Moving your mouse over your admiral name shows the admiral experience",
+    "The resource icons will glow if it is regenerating",
+    "Aircraft Proficiency is included in the displayed FP. Find the detail by moving your mouse over it",
+    "2-5 Fall formula is used by default to calculate LoS. Other formulas are available by moving your mouse over it",
+    "Press the Boss Key to hide the window instantly and bring it back! The default shortcut is Ctrl/Cmd+i",
+    "Quest counters are saved locally on your computer. To let them work correctly, you have to keep using the same computer",
+    "You can zoom out the display in the seetings if you feel your screen is too small",
+    "Switch on \"Warn before quitting\" in settings to avoid closing poi by mistake",
+    "If the game gets stuck regularly (esp. after a big update of the game), you prob'ly need to clear the cache",
+    "",
+    "You can switch off certain notifications in settings. Click the arrow to its right for detailed settings",
+    "If you're having trouble downloading plugins, try switching to other npm sources in \"Plugin/Advanced\"",
+    "View the fleet composition with complete equipment names in plugin \"Composition record\"",
+    "In plugin \"Prophet\", turn on/off the display of ship levels with double click",
+    "Damage during aerial combats is not included in the MVP prediction in plugin \"Prophet\" because it is not provided by the server",
+    "The data of plugin \"Map HP\" are updated after you choose \"Sortie\" in the game",
```

Comment:
I see.
"choose" means "pick out or select from *two or more alternatives*". In this context, there's no alternative. We may use "click" instead of "choose"